### PR TITLE
(maint) Move windows nuget packaging artifacts

### PR DESCRIPTION
**Pull request by melissa**
This commit moves the windows nuget packaging artifacts from
templates/windows to resources/windows/nuget. We have more then just
templates that we need to access as a part of the packaging process. The
main directory name change reflects that. As we get ready to add support
for building MSI packages, isolating nuget specific packaging artifacts
will help maintain clarity in our available resources.

### CODE REVIEW

File: lib/vanagon/platform/windows.rb [hunk 1]
```diff
@@ -92,7 +92,7 @@ def generate_msi_package(project)
         #   -dr             - Directory reference to root directories (cannot contains spaces e.g. -dr MyAppDirRef)
         #   -sreg           - Suppress registry harvesting.
         #   -var <variable> - Substitute File/@Source="SourceDir" with a preprocessor or a wix variable
-        "cd $(tempdir)/#{project.name}; \"$$WIX/bin/heat.exe\" dir staging -v -ke -indent 2 -cg #{cg_name} -gg -dr #{dir_ref} -sreg -var var.StageDir -out wix/#{project.name}-harvest.wxs",
+        "cd $(tempdir)/#{project.name}; \"C:/Program Files (x86)/WiX Toolset v3.10/bin/heat.exe\" dir staging -v -ke -indent 2 -cg #{cg_name} -gg -dr #{dir_ref} -sreg -var var.StageDir -out wix/#{project.name}-harvest.wxs",
```
Comment (MikaelSmith) [inline lib/vanagon/platform/windows.rb]: Do we really need to hard-code the WIX path?

Comment (melissa) [inline lib/vanagon/platform/windows.rb]: OOps, that wasn't supposed to be in this PR...

Comment (underscorgan) [inline lib/vanagon/platform/windows.rb]: How does this relate to the other changes in here?

**Comment by underscorgan** [comment]
:+1: pending travis

**Comment by MikaelSmith** [comment]
:+1:

**Pull request merged**